pym/bob/input.py

```diff
@@ -576,6 +577,62 @@ def isDeterministic(self):
     def hasJenkinsPlugin(self):
         return True
 
+    def callGit(self, workspacePath, *args):
+        cmdLine = ['git']
+        for arg in args:
+            cmdLine.append(arg)
+        try:
+            output = subprocess.check_output(cmdLine, cwd=os.path.join(os.getcwd(),workspacePath, self.__dir),
+                universal_newlines=True, stderr=subprocess.STDOUT)
```

Comment:
Redirecting stderr to stdout give stdout back as `e.output`
```python
 except subprocess.CalledProcessError as e:
            raise BuildError("git error: '{}' '{}'".format(" ".join(cmdLine), e.output))
```
This way the output is colorized like the build error and prefixed with `Build error: git error: 'git status --porcelain' 'error: <stderr>`

Otherwise stderr is white and BuildError is output after git's stderr.